docs/http_api.rst

```diff
@@ -252,16 +252,21 @@ Messages
    Send an outbound (mobile terminated) message.
 
    :param str to:
-       the address (e.g. MSISDN) to send the message too. Should be omitted
-       if ``reply_to`` is specified.
+       the address (e.g. MSISDN) to send the message too.
+       If Junebug is configured with ``allow_late_replies`` The ``to``
```

Comment:
The config seems to suggest that this should be `allow_expired_replies` instead?